ethereum/contracts/IncentivizedOutboundChannel.sol

```diff
@@ -3,14 +3,33 @@ pragma solidity >=0.7.6;
 pragma experimental ABIEncoderV2;
 
 import "./OutboundChannel.sol";
+import "./DOTApp.sol";
 
 // IncentivizedOutboundChannel is a channel that sends ordered messages with an increasing nonce. It will have incentivization too.
 contract IncentivizedOutboundChannel is OutboundChannel {
 
-    constructor() {
+    uint256 public relayFee;
+    address public feeController;
+    DOTApp private _DOTApp;
+
+    constructor(uint256 _relayFee, address _feeControllerAddress) {
```

Comment:
I'd prefer if we have zero constructor parameters for now, mostly due to the current structure of our unit tests.

I suggest defaulting `_relayFee` to 0, and `_feeControllerAddress` to `msg.sender`.

Reply:
Ah ok I see you've updated the unit tests at https://github.com/Snowfork/polkadot-ethereum/pull/294/files#diff-5a92558b750f5ec20439a92fe71c62cbbf40115e3b721e8d8ce106d9d3a336baR167, so feel free to ignore this.